File: app/src/androidTest/java/pavelnazimok/uitestingfeatures/java/tests/SecondScreenTest.java
package pavelnazimok.uitestingfeatures.java.tests;

import androidx.test.ext.junit.runners.AndroidJUnit4;

import org.junit.Test;
import org.junit.runner.RunWith;

@RunWith(AndroidJUnit4.class)
public class SecondScreenTest extends BaseTest {

    @Test
    public void alertDialog_afterClickOk_shouldNotBeDisplayed() {
        menuRobot
                .clickSecondScreen();

        secondScreenRobot
                .clickAlertDialog()
                .clickOkInAlertDialog()
                .assertAlertDialogIsNotDisplayed();
    }
}

File: app/src/androidTest/java/pavelnazimok/uitestingfeatures/kotlin/tests/SecondScreenTest.kt
package pavelnazimok.uitestingfeatures.kotlin.tests

import org.junit.Test
import pavelnazimok.uitestingfeatures.kotlin.robots.onMenu
import pavelnazimok.uitestingfeatures.kotlin.robots.onSecondScreen

class SecondScreenTest : BaseTest() {

    @Test
    fun alertDialog_afterClickOk_shouldNotBeDisplayed() {
        onMenu {
            clickSecondScreen()
        }

        onSecondScreen {
            clickAlertDialog()
            clickOkInAlertDialog()
            assertAlertDialogIsNotDisplayed()
        }
    }
}
